samples/x-nb-psc-mig-l7xlb/README.md

```diff
@@ -0,0 +1,100 @@
+# Apigee X exposed in Multiple GCP Regions with PSC and MIG External HTTPS Load Balancer
+
+End-to-end sample terraform code to provision Apigee X exposed by External Load Balancer with PSC and MIG.
+
+If you plan to use PSC for Apigee northbound network routing, follow the instructions in this document to configure active health check. 
+At this time, PSC does not support active health check monitoring. To work around this limitation of PSC, you can modify the Apigee installation configuration to use a managed instance group (MIG), which does provide active health check capability. Refer the network diagram below
```

Comment:
Minor detail but in my understanding the missing health check is a problem of PSC-NEGs not the PSC. Can we maybe also link to this [doc](https://cloud.google.com/load-balancing/docs/negs) and add a link to this solution from the NB PSC with Neg Example?